components/homme/CMakeLists.txt

```diff
@@ -314,15 +314,15 @@ MESSAGE(STATUS "CXX Flags = ${CMAKE_CXX_FLAGS}")
 MESSAGE(STATUS "Linker Flags = ${CMAKE_EXE_LINKER_FLAGS}")
 
 SET (HOMMEXX_ENABLE_GPU FALSE)
-SET (HOMMEXX_ENABLE_GPU_F90 FALSE) 
```

Comment:
For as much as I agree with these changes in Homme, this seems unrelated, and will cost you a wait for nightly testing. Imho, it could be done separately.

Reply:
You could just leave the `XYZ_BUILD` everywhere, and replace them with `Kokkos_ENABLE_XYZ` in `src/physics/rrtmgp/CMakeLists.txt`.

Reply:
True. I will rework the PR once I can figure out why it's not passing CI.